storage/container_storage_interface/persistent-storage-csi-ebs.adoc

```diff
@@ -1,51 +1,51 @@
 :_content-type: ASSEMBLY
 [id="persistent-storage-csi-ebs"]
-= AWS Elastic Block Store CSI Driver Operator
+= Amazon Elastic Block Store CSI Driver Operator
```

Comment:
Revoked.

Reply:
@smunje1 It doesn't seem like this title is linking out to OCP resources or code that reference "AWS EFS". "Amazon EFS CSI driver" is the appropriate name to be used for the driver. Is there a reason I'm missing why this update should be revoked?

https://docs.aws.amazon.com/eks/latest/userguide/efs-csi.html

Reply:
Revoked. As the service name is associated with operator.